src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -1829,7 +1829,7 @@ internal void ParseLinkHeader(HttpResponseMessage response, System.Uri requestUr
 
             // we only support the URL in angle brackets and `rel`, other attributes are ignored
             // user can still parse it themselves via the Headers property
-            string pattern = "<(?<url>.*?)>;\\s*rel=\"(?<rel>.*?)\"";
+            string pattern = "<(?<url>.*?)>;\\s*rel=(\"?)(?<rel>[^\"]*)\\2";
```

Comment:
Looking definition in https://tools.ietf.org/html/rfc5988#section-5 I think it should be:
```
<(?<url>.*?)>;\s*rel=("?)(?<rel>.*?)\1[^\w-.]?
```

Reply:
thanks but it won't match this case `Link: <https://example.com>; rel=preconnect`

Reply:
It seems your pattern is not for .Net. You can test in PowerShell with ([regex]"pattern").IsMatch("teststring").

Also please consider that if no quotes is used we should search a terminator that is not "alfanumeric, dash and point".